tests/utils.go

```diff
@@ -4482,7 +4482,7 @@ func waitForConfigToBePropagatedToComponent(podLabel string, resourceVersion str
 			}
 		}
 		return true
-	}, 10*time.Second, 1*time.Second).Should(BeTrue(), "Not all kubevirt components picked up the kubevirt config successfully")
+	}, 300*time.Second, 1*time.Second).Should(BeTrue(), "Not all kubevirt components picked up the kubevirt config successfully")
```

Comment:
Can you please explain what is the reason for increasing the timeout and how it is related to this PR?